addrmgr/network.go

```diff
@@ -121,29 +121,12 @@ func isOnionCatTor(netIP net.IP) bool {
 type NetAddressType uint8
 
 const (
-	LocalAddress NetAddressType = iota
+	UnknownAddressType NetAddressType = iota
```

Comment:
This (and I presume other changes in upcoming commits) is a breaking change to the public API, so a new major version of the module will be needed before this PR can be merged.